frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -79,8 +79,10 @@ function PropertyPaneContents({
                         operator={operator}
                         value={value}
                         onChange={(newOperator, newValue) => {
-                            onComplete()
                             setThisFilter(propkey, newValue, newOperator, type)
+                            if (newOperator && newValue) {
```

Comment:
Q: does this work for flags?

Reply:
yup it actually is better now because when you switch from flags to nonflag operator it won't close the selector until you have a value selected